frescobaldi_app/main.py

```diff
@@ -189,6 +191,9 @@ def main():
         sys.stdout.write(debuginfo.version_info_string() + '\n')
         sys.exit(0)
 
+    if args.python_ly:
+        sys.path.insert(1, args.python_ly)
```

Comment:
@uliska Can you add a comment here? I would copy the corresponding commit message:
"The path to python-ly has to be inserted at the *second* position
because the first element in sys.path is the directory of the invoked
script (an information we need in determining if Frescobaldi is run from
its Git repository)"